### DISCUSSION TITLE
how to use omitEmptyAndOptionalProperties feature in .net

### DISCUSSION THREAD

**Discussion by ahmed-abdelrazek** [scalar/scalar#3971] [Q&A]
i was looking forward for this feature https://github.com/scalar/scalar/pull/3899 but i can't find anything about how to enable it in asp core i'm on latest version 1.2.41

**Comment by hanspagel** [comment]
You can’t configure this (as of now), would you mind to further elaborate what you try to achieve?

**Comment by ahmed-abdelrazek** [comment]
i have optional properties in json body that doesn't show in request body when i want to test a request and i have to copy or type them manually
![image](https://github.com/user-attachments/assets/6d2727ec-0c4e-4a87-b7e6-0d9829dc7d2e)
never mind the latest version seems to enable this by default because all my optional properties now show up

wasn't in the body before 
![image](https://github.com/user-attachments/assets/f5a4b08f-fd34-415b-916c-0ec42e994bc1)
